src/Controllers/SchemaController.js

```diff
@@ -660,6 +666,13 @@ const _AudienceSchema = convertSchemaToAdapterSchema(
     classLevelPermissions: {},
   })
 );
+const _IdempotencySchema = convertSchemaToAdapterSchema(
+  injectDefaultSchema({
+    className: '_Idempotency',
+    fields: defaultColumns._Idempotency,
+    classLevelPermissions: {},
```

Comment:
@dplewis can you confirm that an empty {} means "Master Key Only"? if we start to cache responses later on this would be a terrible class to not have locked down.

Reply:
I think all “special” classes are master key only via the router / middleware. That should already be implemented in this PR.

Reply:
Note: This PR does not even introduce a router for the internal `_Idempotency` collection. The collection is designed to be inaccessible.